isis/src/control/objs/ControlNetDiff/ControlNetDiff.cpp

```diff
@@ -88,14 +87,31 @@ namespace Isis {
    * @return Pvl The collection of all differences
    */
   Pvl ControlNetDiff::compare(FileName &net1Name, FileName &net2Name) {
+
+    ControlNetVersioner cnv1(net1Name);
+    ControlNetVersioner cnv2(net2Name);
+
+    return compare(cnv1, cnv2, net1Name, net2Name);
+  }
+
+
+  /**
+   * Compare two Control Networks in ControNetVersioner form, and return their
+   * differences.
+   *
+   * @param cnv1 first Control Network
+   * @param cnv2 second Control Network
+   * @param net1Name FileName of the first Control Network
+   * @param net2Name FileName of the second Control Network
+   * @return Pvl The collection of all differences
+   */
+  Pvl ControlNetDiff::compare(ControlNetVersioner &cnv1, ControlNetVersioner &cnv2,
+                              FileName &net1Name, FileName &net2Name) {
```

Comment:
I don't really understand the necessity for this change. What does exposing the new function do that the old signature prevented?

Reply:
You're right. Major brain lock on my part. Reverted back to the original ControlNetDiff.cpp;h.